src/DotLiquid/StandardFilters.cs

```diff
@@ -650,6 +650,42 @@ public static object Round(object input, object places = null)
             }
         }
 
+        /// <summary>
+        /// Rounds a decimal value up to the next integer, unless already the integer value, removing all decimal places 
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns>The rounded value; null if an exception have occured</returns>
+        public static object Ceil(object input)
+        {
+            try
+            {
+                var i = Convert.ToDecimal(input);
+                return Math.Ceiling(i);
+            }
+            catch (Exception)

```

Comment:
do not swallow all exceptions. target specific exception you need handling of, such as conversion exception, TBD if any math out of range exception would apply here at the edge. You don't want to swallow a memory exception or the like by accident here.

Reply:
Ok - I copied this from Round() and modified from there, so I will remove the try/catch as there is no math exception I would catch that should not bubble up to the calling program (i.e. you try to Ceil(Int64.MaxValue) or something really weird should probably fire an exception).

Reply:
Yes for those specific case you want to catch the specific exception
Console.WriteLine(Math.Ceiling(Decimal.MaxValue)); however does not throw
https://dotnetfiddle.net/MsJ4Kq

Reply:
Round(...)'s precedence... not great :-) should fix it if we have the time, it will be tricky to figure out all exceptions it used to catch to weed the good from the bad.